### PR TITLE
Update configure-runasusername.md

### PR DESCRIPTION
**Pull request by guzj11**

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md [hunk 1]
```diff
@@ -120,7 +111,7 @@ To specify the username with which to execute a Container's processes, include t
 
 ## 为容器设置 Username
 
-要指定运行容器时所使用的用户名，请在容器清单中包含 `securityContext` （[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)）字段，并在其内部包含 `windowsOptions` （[WindowsSecurityContextOptions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)）字段的 `runAsUserName` 字段。
+要指定运行容器时所使用的用户名，请在容器清单中包含 `securityContext` （[SecurityContext](/zh/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)）字段，并在其内部包含 `windowsOptions` （[WindowsSecurityContextOptions](/zh/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)）字段的 `runAsUserName` 字段。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-runasusername.md:114]:
```suggestion
要指定运行容器时所使用的用户名，请在容器清单中包含 `securityContext`
（[SecurityContext](/zh/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)）
字段，并在其内部包含 `windowsOptions`
（[WindowsSecurityContextOptions](/zh/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)）
字段的 `runAsUserName` 字段。
```

Comment (guzj11) [reply content/zh/docs/tasks/configure-pod-container/configure-runasusername.md]: fixed

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md [hunk 2]
```diff
@@ -12,22 +12,13 @@ weight: 20
...
-
-<!--
-This feature is in beta. The overall functionality for `RunAsUserName` will not change, but there may be some changes regarding the username validation.
--->
-{{< note >}}
-此功能目前处于 Beta 状态。`RunAsUserName` 的整体功能不会出现变更，
-但是关于用户名验证的部分可能会有所更改。
-{{< /note >}}
+本页展示如何为运行为在 Windows 节点上运行的 pod 和容器配置 `RunAsUserName` 。
+大致相当于 Linux 上的 `runAsUser`，允许在容器中以与默认不同的 username 运行应用。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-runasusername.md:21]:
```suggestion
大致相当于 Linux 上的 `runAsUser`，允许在容器中以与默认值不同的用户名运行应用。
```

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md [hunk 3]
```diff
@@ -12,22 +12,13 @@ weight: 20
...
-用户名运行容器 entrypoint。
-
-<!--
-This feature is in beta. The overall functionality for `RunAsUserName` will not change, but there may be some changes regarding the username validation.
--->
-{{< note >}}
-此功能目前处于 Beta 状态。`RunAsUserName` 的整体功能不会出现变更，
-但是关于用户名验证的部分可能会有所更改。
-{{< /note >}}
+本页展示如何为运行为在 Windows 节点上运行的 pod 和容器配置 `RunAsUserName` 。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-runasusername.md:20]: pod -> Pod

**Review by tengqm — COMMENTED** [review]
lgtm other than two nits

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md (hunk 3 above)
Comment (guzj11) [reply content/zh/docs/tasks/configure-pod-container/configure-runasusername.md]: Fixed

File: content/zh/docs/tasks/configure-pod-container/configure-runasusername.md (hunk 2 above)
Comment (guzj11) [reply content/zh/docs/tasks/configure-pod-container/configure-runasusername.md]: Fixed

**Pull request merged**